lib/pigeon/adm.ex

```diff
@@ -120,9 +120,9 @@ defmodule Pigeon.ADM do
   defp cast_push(worker_name, notifications, on_response) when is_list(notifications) do
     for n <- notifications, do: cast_push(worker_name, n, on_response)
   end
-  defp cast_push(worker_name, notification, nil) do
-    GenServer.cast(worker_name, {:push, :adm, notification})
-  end
+  #defp cast_push(worker_name, notification, nil) do
+  #  GenServer.cast(worker_name, {:push, :adm, notification})
+  #end
```

Comment:
I don't *think* this is used anymore, but not sure.

Reply:
Looks like you are correct, can be deleted.

Reply:
Remember to also delete the corresponding unused clause of `handle_cast/2` in ADM.Worker line 38